src/swipe.js

```diff
@@ -0,0 +1,109 @@
+import {addListenerMulti} from './utilities';
```

Comment:
Since this is more or less angular's, $swipe we should want to have some comments telling where this code came from.